src/components/templates/PartyMap/components/Map/MapRoom.scss

```diff
@@ -12,9 +12,17 @@ $label-visible: flex;
   flex-direction: column;
   align-items: center;
 
+  appearance: none;
+  -webkit-appearance: none;
+  background: none;
+  border: none;
```

Comment:
Why are these being set to `none`?

Reply:
Same inheritance issue after changing tag